### ISSUE TITLE
Fusion Core bugging after breaking at least one electromagnet

### ISSUE THREAD

**Issue by Dubl33** [turbodiesel4598/NuclearCraft#561]
When I tried placing cooler to test cooling with gelid cryotheum, I accidentally broke 1 electromagnet and all the pink fluid got out of the fusion reactor, when I put back the electromagnet, the reactor core started heating up back to 8k kK and when it got there (with power provided the whole time and electromagnets powered), it was just going between 7990 kK and 8010 kK without the reactor starting up (I may add that it had everything it needed to start up). If I save and quit and load the world back up, it starts working (note: If there is no fuel in it after loading the world back up then you need to put fuel in it and save and quit and load back up again)
I am playing enigmatica 2 expert mode with the mod version 2.16f-1.12.2

**Comment by derdotte** [comment]
I can only reproduce aboves bug by inserting power into the reactor core before the structure is complete, the setup is simple:  
I doubt toroid size matters but for this i used toroid size 3:
I doubt that fuel matters but i am using deuterium/tritium.
Place the structure, but destroy an electromagnet, heat up the core by your favorite infinite power source (using RFtools creative powercell).
Have it at ignition temperature, add the destroyed electromagnet back and see how the temperature goes above ignition temp and goes down again. Never actually igniting.
I tested this with plasma from a previously succesful ignition (through normal means) and without, both will fail.

**Comment by derdotte** [comment]
I suppose a good way to prevent this from happening is to check wether the structure is verified, if not it should not accept any power. At the end, it already does turn off when the structure is destroyed, so the idea would be to use that trigger to go through this power accepting process too (Is structure verified? if true -> acceptPower = true)

**Comment by turbodiesel4598** [comment]
Thanks for the details! I knew there was some bug like this, but wasn’t exactly sure what was causing it :P

**Comment by turbodiesel4598** [comment]
@Dubl33 @derdotte Turns out I'm not actually able to recreate either of these issues. Are you both using 2.16f, or are you perhaps using the latest version derdotte?

**Comment by derdotte** [comment]
@turbodiesel4598 Currently using NuclearCraft-2.16f-1.12.2.jar its part of the enigmatica 2 expert skyblock modpack. There i was also able to reproduce this multiple times succesfully.

**Comment by turbodiesel4598** [comment]
Could you try updating to the latest version of the mod and see if it happens then?

**Comment by derdotte** [comment]
Doing so right now, gonna update in a few mins.

**Comment by derdotte** [comment]
@turbodiesel4598 yep, the newest version fixes this issue.  
I used the same method as above multiple times and even checked wether it would reignite even if you broke the structure multiple times. I also checked wether if you were to drop below 8kk Kelvin it would reignite after giving it energy prior to fixing the structure, which also worked.
I read the changelog during startup and i cant really see what you fixed between 2.16f and 2.17a.
It still regularly occurs on 2.16f using aboves method though.  

For testing i simply exchanged nuclearcraft versions on a duplicated profile of enigmatica 2 expert skyblock.
